common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -159,6 +162,26 @@ private int donateLink(CommandContext<CommandSourceStack> context) {
         return 1;
     }
 
+    private int simulate(CommandContext<CommandSourceStack> context) {
```

Comment:
Did you intend for this to go into production, or is it debug code that is left behind?

Reply:
Originally, I meant for it to go into production but now thinking about it, there isn't really a way to implement it that would be useful to mod developers. Do you have any ideas? If not, it can be removed.

Reply:
I don't even understand the purpose of this command. Please explain how you think this should be used?

Reply:
I thought that it could be used by developers to test features that needed a chat output, to save time (for example, they could simulate text of them completing a Raid instead of actually doing it,saving a few minutes.

Reply:
I think you'll just have to write such a command yourself if you really need to test it, and then throw it away.
*Possibly* we could introduce some kind of "DebugCommands" similar to DebugFeatures. In any case this kind of stuff to not belong in WynntilsCommand.

Reply:
Removed.